CSS/components/alert.css

```diff
@@ -0,0 +1,28 @@
+.avatar{
+    border-radius: 100%;
+    width: 5rem;
```

Comment:
instead of simple width and height, add min-width and min-height this gives you a chance to make the image circle